docs/website/docs/hub/intro.md

```diff
@@ -3,41 +3,120 @@ title: Introduction
+    C --> D[Deploy Workspace]
+    D --> E[Maintain data quality]
+    E --> F[Share]
 ```
 
-You can try out any features by self-issuing a trial license. You can use such license for evaluation, development and testing.
-Trial license are issued off-line using `dlt license` command:
+In practice, this means any Python developer can:
 
-1. Display a list of available features
-```sh
-dlt license scopes
-```
+* Build and customize data pipelines quickly (with LLM help when desired)
+* Derisk data insights by keeping data quality high with checks, tests, and alerts
+* Ship fresh dashboards, reports, and data apps
+* Scale the data workflows easily without babysitting infra, schema drift, and silent failures
 
-2. Issue license for the feature you want to test.
 
-```sh
-dlt license issue dlthub.transformation
-```
 
-The command above will enable access to new `@dlt.hub.transformation` decorator. Note that you may
-self issue licenses several times and the command above will carry-over features from previously issued license.
+:::tip
+Want to see it end-to-end? Watch the dltHub [Workspace demo](https://youtu.be/rmpiFSCV8aA).
+:::
+
+To get started quickly, follow the [installation instructions](getting-started/installation.md)
+
+## Overview
+
+### Key capabilities
+
+1. **[LLM-native workflow](../dlt-ecosystem/llm-tooling/llm-native-workflow)**: accelerate pipeline authoring and maintenance with guided prompts and copilot experiences.
+
+2. **[Transformations](features/transformations/index.md)**: write Python or SQL transformations with `@dlt.hub.transformation`, orchestrated within your pipeline.
+
+3. **[Data quality](features/quality/data-quality.md)**: define correctness rules, run checks, and fail fast with actionable messages.
+
+4. **[Data apps & sharing](../general-usage/dataset-access/marimo)**: build lightweight, shareable data apps and notebooks for consumers.
+
+5. **[AI agentic support](features/mcp-server.md)**: use MCP servers to analyze pipelines and datasets.
+6. **Managed runtime**: deploy and run with a single command—no infra to provision or patch.
+7. **[Storage choice](ecosystem/iceberg.md)**: pick managed Iceberg-based lakehouse, DuckLake, or bring your own storage.
+
+### How dltHub fits with dlt (OSS)
+
+dltHub embraces the dlt library, not replaces it:
+* dlt (OSS): Python library focused on extract & load with strong typing and schema handling.
+* dltHub: Adds transformations, quality, agentic tooling, managed runtime, and storage choices, so you can move from local dev to production seamlessly.
+
+If you like the dlt developer experience, dltHub gives you everything around it to run in production with less toil.
+
+## dltHub products
+dltHub consists of three main products. You can use them together or compose them based on your needs.
+
+### Workspace
+
+**[Workspace](workspace/overview.md)** - the unified environment for building, running, and maintaining data workflows end-to-end.
```

Comment:
All those should have availability indicatior like "In Preview"

Reply:
added the indicator